frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsagePlanSummary.tsx

```diff
@@ -31,94 +44,145 @@ const StyledCardDescription = styled(Box)(({ theme }) => ({
+                    </StyledColumnGrid>
+                </StyledContainer>
+            </StyledGrid>
+            <ConditionallyRender
+                condition={
+                    estimateFlagEnabled && includedTraffic > 0 && overages > 0
+                }
+                show={
+                    <StyledGrid item xs={5.5} md={5.5}>
+                        <StyledContainer>
+                            <StyledColumnGrid item>
+                                <StyledCardTitleRow>
+                                    <b>Accrued traffic charges</b>
+                                </StyledCardTitleRow>
+                                <StyledCardDescription>
+                                    <RowContainer>
+                                        Requests overages this month (
+                                        <Link href='https://www.getunleash.io/pricing'>
+                                            pricing
+                                        </Link>
+                                        )
+                                        <StyledNumbersDiv>
+                                            {overages.toLocaleString()} requests
+                                        </StyledNumbersDiv>
+                                    </RowContainer>
+                                    <RowContainer>
+                                        Accrued traffic charges
+                                        <StyledNumbersDiv>
+                                            <Badge color='secondary'>
+                                                {overageCost} USD
+                                            </Badge>
+                                        </StyledNumbersDiv>
+                                    </RowContainer>
+                                    <ConditionallyRender
+                                        condition={estimatedMonthlyCost > 0}
+                                        show={
+                                            <RowContainer>
+                                                Estimated traffic charges based
+                                                on current usage
+                                                <StyledNumbersDiv>
+                                                    <Badge color='secondary'>
+                                                        {estimatedMonthlyCost}{' '}
+                                                        USD
+                                                    </Badge>
+                                                </StyledNumbersDiv>
+                                            </RowContainer>
+                                        }
+                                    />
+                                </StyledCardDescription>
+                            </StyledColumnGrid>
+                        </StyledContainer>
+                    </StyledGrid>
+                }
+            />
+        </StyledContainerGrid>
```

Comment:
Might just be me, but from a design perspective it's a bit weird to me that some of these are badges but not others.  I recommend double checking with @FredrikOseberg just in case.

![image](https://github.com/Unleash/unleash/assets/14320932/5ef8d32c-0dcb-40b5-be60-135b78365c3a)